helm-chart-sources/pulsar/values.yaml

```diff
@@ -1752,6 +1752,31 @@ pulsarHeartbeat:
       topicName: "persistent://public/default/pubsub-latency-test"
   service: {}
 
+# For configuring OpenID in Okta for Pulsar, instructions are here: https://www.youtube.com/watch?v=UQBrecHOXxU&ab_channel=DataStaxDevelopers
+# Please also see the docs here: https://github.com/datastax/pulsar-openid-connect-plugin
+# Then, use the values from the Payload generated to populate these fields:
+openid:
```

Comment:
Please add a key value pair here for the `withS4K` defaults. That will give an indication of what is available.